pkg/ctl/install/install.go

```diff
@@ -83,8 +83,8 @@ func installFluxCmd(rc *cmdutils.ResourceCmd) {
 			"Email to use as Git committer")
 		fs.StringVar(&opts.gitFluxPath, "git-flux-subdir", "flux/",
 			"Directory within the Git repository where to commit the Flux manifests")
-		fs.DurationVar(&opts.timeout, "timeout", 20*time.Second,
-			"Timeout for I/O operations")
+		fs.DurationVar(&opts.timeout, "git-timeout", 20*time.Second,
```

Comment:
I guess it could also be the timeout for kubernetes operations?

Reply:
I'd rather not expose multiple timeout flags, there will be too many. Let's extract `--timeout` from `AddCommonFlagsForAWS` and provide alternative default value and description.

Reply:
As clarified offline with Ilya, I'll go for the following:
```go
func AddCommonFlagsForAWS(group *NamedFlagSetGroup, p *api.ProviderConfig, cfnRole, timeout bool) {
	group.InFlagSet("AWS client", func(fs *pflag.FlagSet) {
// [...]
		if timeout {
			AddTimeout(fs, &p.WaitTimeout, api.DefaultWaitTimeout, "max wait time in any polling operations")
		}
// [...]
	})
}

// AddTimeout adds a timeout flag with the provided value and description.
func AddTimeout(fs *pflag.FlagSet, p *time.Duration, value time.Duration, usage string) {
	fs.DurationVar(p, "timeout", value, usage)
}
```
We can then pass `false` to `AddCommonFlagsForAWS` in the case of `eksctl install flux` and call `AddTimeout` separately.

Reply:
I actually changed my mind about this. See: https://github.com/weaveworks/eksctl/pull/1082#discussion_r311913745